### PR TITLE
docs：Add biweekly report

### PR DESCRIPTION
**Pull request by Qizeng-api7**
Add biweekly report 2023.07.03-2023.07.16

### CODE REVIEW

File: blog/en/blog/2023/07/18/weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+3. Fixed a cache key conflict issue in the etcd watch implementation.
+
+4. Added a new feature to the admin API that allows forceful deletion of resources using the "force" parameter.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/07/17/ndDgARK1_%E6%89%80%E6%9C%89%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
```
Comment (yzeng25) [inline blog/en/blog/2023/07/18/weekly-report.md]: Here is a typo in the poster, please fix it.

image

File: blog/zh/blog/2023/07/18/weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   
+3. 修复 etcd watch 实现中 update_count 冲突问题
+   
+4. admin API 增加通过 force 参数强制删除资源特性
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/07/17/ndDgARK1_%E6%89%80%E6%9C%89%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
```
Comment (yzeng25) [inline blog/zh/blog/2023/07/18/weekly-report.md]: ditto, please fix the typo in the poster

**Comment by yzeng25** [comment]
@Qizeng-api7 , same typo is found in multiple blogs, please see https://github.com/apache/apisix-website/issues/1626 and update the blogs accordingly.

**Review by Yilialinn — APPROVED** [review]

**Comment by Qizeng-api7** [comment]
I will gradually fix the typos.

File: blog/zh/blog/2023/07/18/weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/07/17/shxmVhkq_%E8%8B%B1%E6%96%87%E5%A4%B4%E5%9B%BE.png
+---
+
+> 我们近期对 Apache APISIX 和 Apache APISIX Ingress Controller 的部分功能进行了修复或改善，包括优化 mock 插件、修复 limit-count 插件、解决缓存键冲突、增加通过 force 参数进行强制删除的功能，详情请阅读本期双周报。与此同时，我们还发布了 Apache APISIX 3.4.0。它的功能更加强大，性能更加优越。
```
Comment (Yilialinn) [inline blog/zh/blog/2023/07/18/weekly-report.md]:
```suggestion
> 我们近期对 Apache APISIX 的部分功能进行了修复或改善，包括优化 mock 插件、修复 limit-count 插件、解决缓存键冲突、增加通过 force 参数进行强制删除的功能，详情请阅读本期双周报。与此同时，我们还发布了 Apache APISIX 3.4.0。它的功能更加强大，性能更加优越。
```

Comment (Yilialinn) [reply blog/zh/blog/2023/07/18/weekly-report.md]: It's not a template. Please check every time. @Qizeng-api7

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**